lib/foreman/exception.rb

```diff
@@ -50,6 +50,9 @@ class MaintenanceException < Foreman::Exception
   class PermissionMissingException < Foreman::Exception
   end
 
+  class LdapException < Foreman::Exception
```

Comment:
Could you please use `Foreman::WrappedException` as a parent for this class? this allows us also to keep the original error and its trace. This means we should move the definition from app/services/foreman/wrapped_exception.rb to this file (or the other way around).